packages/uploadthing/src/internal/handler.ts

```diff
@@ -149,21 +85,12 @@ type UploadThingResponse = {
   key: string;
 }[];
 
-export const buildRequestHandler = <
-  TRouter extends FileRouter,
-  TRuntime extends AnyRuntime,
->(
+export const buildRequestHandler = <TRouter extends FileRouter>(
   opts: RouterWithConfig<TRouter>,
 ) => {
   return async (input: {
     req: RequestLike;
-    res?: TRuntime extends "pages"
-      ? NextApiResponse
-      : TRuntime extends "express"
-      ? ExpressResponse
-      : TRuntime extends "fastify"
-      ? FastifyReply
-      : undefined;
+    res?: unknown;
```

Comment:
doesn't really make a lot of sense that this is typed, it's just in the way similar to #359